specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/stable/2021-04-01/examples/ResourceChangesList.json

```diff
@@ -0,0 +1,68 @@
+            "name": "ARG_23fa00fd-dda0-4268-b482-2076825cf165_970d8c6d-6b78-4270-92ef-88d5aa2b5f0b_132316363294700000_132316498613900000",
+            "type": "Microsoft.ChangeAnalysis/resourceChanges",
+            "properties": {
+              "resourceId": "/subscriptions/4d962866-1e3f-47f2-bd18-450c08f914c1/resourceGroups/MyResourceGroup/providers/Microsoft.Web/sites/mysite",
+              "timeStamp": "2021-04-26T02:17:41.39Z",
+              "initiatedByList": [
+                "ellen@contoso.com"
+              ],
+              "changeType": "Update",
+              "propertyChanges": [
+                {
+                  "changeType": "Update",
+                  "changeCategory": "User",
+                  "jsonPath": "value[1].properties.thumbprint",
+                  "displayName": "publicCertificates[\"AppCert\"].properties.thumbprint",
+                  "level": "Important",
+                  "description": "The thumbprint of the certificate",
+                  "oldValue": "21D0482F-E91E-4C14-8078-65BFDCDBCA64",
+                  "newValue": "3F2DF554-B063-4383-8BD3-4970BCF20A7E",
+                  "isDataMasked": false
+                }
+              ]
+            }
+          },
+          {
+            "id": "/subscriptions/4d962866-1e3f-47f2-bd18-450c08f914c1/resourceGroups/MyResourceGroup/providers/Microsoft.Web/sites/mysite/extensions/AppStateTracker/providers/Microsoft.ChangeAnalysis/resourceChanges/AST_b2ecd7ec-00dd-4d44-bd19-fefc88790c05_da36e22e-f541-44bd-8d89-8f10a27af4ad_132592897881782408_132592957873393845",
+            "name": "AST_b2ecd7ec-00dd-4d44-bd19-fefc88790c05_da36e22e-f541-44bd-8d89-8f10a27af4ad_132592897881782408_132592957873393845",
+            "type": "Microsoft.ChangeAnalysis/changes",
+            "properties": {
+              "resourceId": "/subscriptions/4d962866-1e3f-47f2-bd18-450c08f914c1/resourceGroups/MyResourceGroup/providers/Microsoft.Web/sites/mysite",
+              "timeStamp": "2021-04-26T02:17:41.39Z",
+              "initiatedByList": [
+                "ellen@contoso.com"
+              ],
+              "changeType": "Update",
+              "propertyChanges": [
+                {
+                  "changeType": "Update",
+                  "changeCategory": "User",
+                  "jsonPath": "environment.environmentVariables.APPSETTING_DB_CONNSTR",
+                  "displayName": "APPSETTING_DB_CONNSTR",
+                  "level": "Important",
+                  "description": "Application setting",
+                  "oldValue": "<old database connection string>",
+                  "newValue": "<new database connection string>",
+                  "isDataMasked": false
```

Comment:
>false [](start = 34, length = 5)

Please show masked values and set it to true.

Reply:
Switched to POST method and will show unmasked value in the example